src/scenario.c

```diff
@@ -817,7 +817,7 @@ int scenario_get_num_packed_objects_to_write()
 	rct_object_entry_extended *entry = (rct_object_entry_extended*)0x00F3F03C;
 
 	for (i = 0; i < 721; i++, entry++) {
-		if (RCT2_ADDRESS(0x009ACFA4, uint32)[i] == 0xFFFFFFFF || (entry->flags & 0xF0))
+		if (RCT2_ADDRESS(RCT2_ADDRESS_RIDE_ENTRIES, uint32)[i] == 0xFFFFFFFF || (entry->flags & 0xF0))
```

Comment:
0x9ACFA4 is old format we should avoid using this. `GET_RIDE_ENTRY(i)` is preferred.